### DISCUSSION TITLE
FFA Scoring

### DISCUSSION THREAD

**Discussion by Dliwk** [efroemling/ballistica#546] [Ideas]
Currently if more than one player scores an equal number of points, all they get as much series points, as one player could get with such score:
| Player | Game Score | Series Score |
|----------|------------------|-------------------|
| Spaz   | 24                |   +8                |
| Kronk  | 0                  |   +4                |
| Zoe     | 0                  |   +4                |
| Mel     | 0                   |   +4                |
| Jack    | -1                 |   +2                |

Which plays against players who win without letting other players gain any points.

Alternative: give each of them as much points, as could get lowest of them:
| Player | Game Score | Series Score |
|----------|------------------|-------------------|
| Spaz   | 24                |   +8                |
| Kronk  | 0                  |   +1 (could be +4) |
| Zoe     | 0                  |   +1 (could be +2) |
| Mel     | 0                   |   +1 (could be +1) |
| Jack    | -1                 |   +0                |

**Comment by TheMikirog** [comment]
For a party game, it feels like a slap in the face, if I give it my all and get barely any points at the end of it, because I tied with someone.
It's much more common to tie with a second person and in general we don't want to extend the session unnecessarily, hence I prefer the old system with getting the maximum amount of points.
If everyone is mediocre, in worst case scenario you can turn a 20 minute session into a 1 hour session if ties happen multiple times.
People in the back not being able to catch up is not the issue with the scoring during ties, it's an issue of peoples' gaming abilities.
So yeah, pacing is my main argument. If I wanted to get into players getting different amounts of series points despite getting the same game score, this comment would be much, **much** longer.

If there's an argument to be made, I think we should focus on score distribution, like how many points players get when 2nd, 3rd and 4th.

**Comment by MatiasMGz** [comment]
This would be cruel as hell like Squid Games, XD.
Please not!!!
